tests/test_local_move.py

```diff
@@ -265,5 +265,5 @@ def local_move(_, steps=500):
         return (xs[-1], boxes[-1]), None
 
     assert_no_drift(
-        (x0[-1], boxes[-1]), local_move, num_particles_near_ligand, n_local_resampling_iterations=250, threshold=0.05
+        (x0[-1], boxes[-1]), local_move, num_particles_near_ligand, n_local_resampling_iterations=250, threshold=0.055
```

Comment:
Why did this change? Which (if any) changes in this PR would we expect to break bitwise equivalence with the current version?

Reply:
The change that made this difference was the erfc approximation we make for float32: https://github.com/proteneer/timemachine/blob/master/timemachine/cpp/src/kernels/k_nonbonded_common.cuh#L34-L44